lib/index.js

```diff
@@ -0,0 +1,13 @@
+var run = require('./run');
+var utils = require('./utils');
+
+module.exports = function () {
+  console.error(
```

Comment:
Worth spinning a 7.0.0 branch to remove this now so that removing this isn't forgotten later?